src/compas_rhino/conduits/base.py

```diff
@@ -25,6 +25,21 @@ def __init__(self, refreshrate=1):
 
     @contextmanager
     def enabled(self):
+        """Create a safe context for the conduit with automatic enabling and disabling.
```

Comment:
`safe` is a bit of an ambiguous term here, since our users might not know or care what are the possible unsafe aspects of a conduit, I would err on the side of being verbose and explain more in detail, e.g. "Create a context for the conduit with automatic enabling and disabling, that ensure all resources required for the it are deallocated properly when the conduit is no longer in use." (or something along those lines)